storage/innobase/srv/srv0start.cc

```diff
@@ -3577,9 +3577,6 @@ static void srv_shutdown_page_cleaners() {
   ut_a(srv_shutdown_state.load() == SRV_SHUTDOWN_MASTER_STOP);
   ut_a(!srv_master_thread_is_active());
 
-  ut_ad(buf_flush_active_lru_managers() == srv_buf_pool_instances ||
-        buf_flush_active_lru_managers() == 0);
-
```

Comment:
Where is the code to wait for lru_managers to exit?
It is not right to remove the assert. Please move it to the place after lru cleaners are closed.

It would be nice to have assert  because there might be bugs in the closing logic.

Reply:
https://github.com/tempesta-tech/percona-server/blob/7f7b9a0130ddb1c55dab2f16dab0d522a5a2cfdc/storage/innobase/srv/srv0start.cc#L3590

Reply:
At this point (https://github.com/tempesta-tech/percona-server/blob/7f7b9a0130ddb1c55dab2f16dab0d522a5a2cfdc/storage/innobase/srv/srv0start.cc#L3590), the code is waiting for lru_managers to exit